src/v/rpc/simple_protocol.cc

```diff
@@ -225,7 +228,9 @@ simple_protocol::dispatch_method_once(header h, net::server::resources rs) {
                       .finally([m, l = std::move(l)]() mutable {
                           m->probes.latency_hist().record(std::move(l));
                       });
-                });
+                })
+                // Stop measurement when the process finishes
+                .finally([msr = std::move(msr)] {});
```

Comment:
Shouldn't `msr` be hooked up into `net::server_probe`? To me it looks like no metric is published here. Am I missing something?

Reply:
Removed.